addons/hr_expense/models/hr_expense.py

```diff
@@ -144,6 +144,11 @@ def _default_employee_id(self):
     duplicate_expense_ids = fields.Many2many(comodel_name='hr.expense', compute='_compute_duplicate_expense_ids')  # Used to trigger warnings
     same_receipt_expense_ids = fields.Many2many(comodel_name='hr.expense', compute='_compute_same_receipt_expense_ids')  # Used to trigger warnings
 
+    split_expense_id = fields.Many2one(
```

Comment:
Maybe the field should rather be called split_expense_origin_id, now ? (for clarity)